osquery/tables/system/windows/certificates.cpp

```diff
@@ -203,7 +203,17 @@ void enumerateCertStore(const HCERTSTORE& certStore,
                     certContext->pCertInfo->SerialNumber.pbData +
                         certContext->pCertInfo->SerialNumber.cbData),
         back_inserter(serial));
-    r["serial"] = serial;
+//serial number was in wrong endian, so reversed the string and then swapped 
+//every other number to make each byte in the correct order
+    std::reverse(serial.begin(), serial.end());
```

Comment:
Huh. I had no idea it came out wrong endian. Oddly, this is the second time today I've seen something like this in the windows depths. Anyhow... the internet suggests this code, which feels a little cleaner to me:
```
BYTE *pbName=pCertContext->pCertInfo->SerialNumber.pbData;
CString theString ;
for (int i = pCertContext->pCertInfo->SerialNumber.cbData - 1; i >= 0 ; i--)
{
  CString hex ;
  hex.Format("%02x", pbName[i]) ;
  theString += hex ;
}
```